docs/_pages/extensibility.md

```diff
@@ -275,6 +275,21 @@ internal static class Initializer
 }
 ```
 
+Unfortunately, only a single module initializer is allowed. That's why Fluent Assertions supports its own "module initializer" through the `[AssertionEngineInitializer]` attribute. It can be used multiple times. 
```

Comment:
I can get several module initializers to run 🤨 
Am I missing something?

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/module-initializers
> When one or more valid methods with this attribute are found in a compilation, the compiler will emit a module initializer which calls each of the attributed methods. The calls will be emitted in a reserved, but deterministic order.

Reply:
You're right, it can and it all seems to work. The other caveat is that .NET Framework 4.x doesn't have that attribute. So when the `FluentAssertions.DataSet` package adds a source file to the target test project with a module initializer, it would also have to define the attribute for .NET 4.x. However, [non-SDK projects do not define attributes](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives) like `NETFRAMEWORK` and such.